lib/Serializer/Normalizer/ReferenceLoopNormalizer.php

```diff
@@ -50,4 +50,9 @@ public function supportsNormalization(mixed $data, string $format = null): bool
     {
         return $format === JsonEncoder::FORMAT;
     }
+
+    public function getSupportedTypes(?string $format): array
+    {
+        return ['*' => false];
```

Comment:
LGTM, just wondering, why should it be [not cacheable](https://github.com/symfony/symfony-docs/issues/18042#issuecomment-1555089540:~:text=while%20%60false%60%20means%20that%20the%0A%20%20result%20is%20not%20cacheable) by default?

Reply:
@kingjia90 I disabled it, because we haven't caching before. Do you mean we can activate it?
```suggestion
        return ['*' => true];
```

Reply:
Just out of curiosity, but found this https://symfony.com/doc/current/serializer/custom_normalizer.html#:~:text=result%20is%20not%20cached%20by%20default%20 seems disabled by default. Probably would need a changelog entry to describe the perfomance improvements or keep it disabled

Reply:
I disabled it again. Can be done in a followup if it is possible